### PR TITLE
Add some ICEs

### PR DESCRIPTION
**Pull request by fanninpm**

### CODE REVIEW

File: ices/82250.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#![feature(let_chains)]
```
Comment (JohnTitor) [inline ices/82250.rs:1]: The issue is now closed as a duplicate.

**Review by JohnTitor — CHANGES_REQUESTED** [review]

**Comment by JohnTitor** [comment]
Also, could you put them in ascending order by issue number next time? It will help my review :)

File: ices/82250.rs (hunk 1 above)
Comment (fanninpm) [reply ices/82250.rs]: Resolved in 7bd368b.

**Comment by fanninpm** [comment]
> Also, when committing, could you put them in ascending order by issue number next time? It will help my review :)

For my future reference, this will involve a `git rebase -i master` (with EDITOR=nvim implied), doing a *visual block* selection (CTRL-v), and then `:'<,'>Vissort` ([reference](https://vim.fandom.com/wiki/How_to_sort_using_visual_blocks)).

**Review by JohnTitor — APPROVED** [review]
Thanks!

**Pull request merged**